src/ui/input/Input.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react";
+import { useController } from "react-hook-form";
+import cn from "classnames";
+import Image from "next/image";
+
+import showPassword from "../../assets/pages/signUp/showPassword.svg";
+
+import { IInputProps } from "../../types/common/UiKitProps";
+
+import { ISignUpForm } from "../../types/components/ComponentsTypes";
+
+import styles from "./Input.module.css";
+
+export const Input = ({ label, type, placeholder, autoComplete, subtitle, error, ...props }: IInputProps) => {
+	const { field, fieldState } = useController<ISignUpForm>(props);
+	const [passwordType, setPasswordType] = useState("password");
+	const togglePasswordVisibility = () => setPasswordType(passwordType === "password" ? "text" : "password");
```

Comment:
can you add these 2 types to an exported enum that will be located in src/helpers/Input.ts and use the imported values ​​here?